docker/app/entrypoint.sh

```diff
@@ -1,22 +1,10 @@
 #!/bin/sh
 
-if [ "x$ENVIRONMENT" = "xdevelopment" ]; then
-    /wait
-    RETCODE=$?
-    if [ "$RETCODE" -gt 0 ]; then
-        exit $RETCODE
-    fi
-fi
-
 # rsyslog needs to run so that lfmerge can log to /var/log/syslog
 /etc/init.d/rsyslog start
 
 # run lfmergeqm on startup to clear out any failed send/receive sessions from previous container
-if which /lfmergeqm-background.sh > /dev/null
-then
-    # MUST be run as a background process as it kicks off an infinite loop to run every 24 hours
-    /lfmergeqm-background.sh &
-fi
+/lfmergeqm-background.sh & # MUST be run as a background process as it kicks off an infinite loop to run every 24 hours
```

Comment:
Per our conversation, `which` is never needed since we control the existence of lfmergeqm-background.sh in a different place.